src/test/java/io/confluent/kafkarest/mock/MockTime.java

```diff
@@ -25,11 +25,18 @@
 
   @Override
   public long milliseconds() {
+    // Incrementing currentMs prevents tests from locking up
+    // "waiting" for the time checks in the code.  Without this
+    // the ConsumerManager never completed (it never calls sleep or waitOn)
+    currentMs++;
```

Comment:
The short answer - you're right, I'll revert the change.  

Before I fixed the max bytes bug this was an issue.  With the maxbytes bug fixed the tests no longer lock up.